bytecomp/matching.ml

```diff
@@ -1772,42 +1868,45 @@ let rec split k xs = match xs with
       let xs,y0,ys = split (k-2) xs in
       x0::xs,y0,ys
 
-let zero_lam  = Lconst (Const_base (Const_int 0))
+let zero_lam loc = Lconst (Const_base (Const_int 0), loc)
 
 let tree_way_test loc arg lt eq gt =
   Lifthenelse
-    (Lprim (Pintcomp Clt,[arg;zero_lam], loc),lt,
-     Lifthenelse(Lprim (Pintcomp Clt,[zero_lam;arg], loc),gt,eq))
+    (Lprim (Pintcomp Clt,[arg;zero_lam loc], loc),loc,lt,loc,
+     Lifthenelse(
+       Lprim (Pintcomp Clt,[zero_lam loc;arg], loc),loc,gt,loc,eq,loc),
+     loc)
```

Comment:
What exactly is the complaint here?  (The line was too long.)

Reply:
`loc` appears 10 times.
I'm not saying this is necessarily wrong, I'm just not sure it's right. Anyway you'd be hard press to notice an issue in gdb, since this code is, I believe, only used by the bytecomp compiler. So I didn't look too much into it.

Reply:
This has been improved.